Tests/RxCocoa/DriveManyTests+RxCocoa.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  DriveManyTests+RxCocoa.swift
+//  RxSwiftExt
+//
+//  Created by Matthew Crenshaw on 7/19/18.
+//  Copyright © 2018 RxSwift Community. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxCocoa
+import RxTest
+import RxSwiftExt
+
+class DriveManyTests: XCTestCase {
+    func testDriveMany() {
+        let scheduler = TestScheduler(initialClock: 0, simulateProcessingDelay: false)
+
+        SharingScheduler.mock(scheduler: scheduler) {
+
+            let values = ["one", "two", "three", "four"]
+
+            let observers = [scheduler.createObserver(String.self),
```

Comment:
Can you make a test that confirms that All disposables are disposed ?

Reply:
Can do. I did not test because of CompositeDisposable's existing tests.